### PR TITLE
Rewrite entire "Verify download" tutorial

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md [hunk 1]
```diff
@@ -100,46 +181,64 @@ gpg:          There is no indication that the signature belongs to the owner.
 Primary key fingerprint: 8439 38DF 228D 22F7 B374  2BC0 D94A A3F0 EFE2 1092
 ```
 
-positive
-: This is an example of a ‘good’ signature. GPG is only validating the integrity of the given file. The warning messages indicate that your current GnuPG trust database does not have trust information for the signing key and that, unless you have actually verified and signed one of the public keys belonging to signers of the Ubuntu ISO image signing key, you will get these warnings.
+A **Good** signature means that the checked file was definitely signed by the owner of the keyfile stated (if they didn't match, the signature would be reported as **BAD**. The warning message is simply there to let you know that you have not countersigned the key and it isn't in your list of trusted sources. If you want to know more about signing keys and trust, you can check out the [Ubuntu community GPG wiki page][gpg-wiki].
```
Comment (degville) [inline tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md]: no end to the start bracket.

File: tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md [hunk 2]
```diff
@@ -2,91 +2,172 @@
...
-Duration: 0:04
+### For Windows
+
+If you are using bash on Windows 10 (why on earth not? See [this tutorial][tut-windows-bash]), these tools are part of the default install.
+
+### For macOS
+
+You can install the latest GnuPG using [Homebrew][homebrew]:
+
+```
```
Comment (degville) [inline tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md]: lots of these are missing *bash*

File: tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md [hunk 3]
```diff
@@ -2,91 +2,172 @@
...
+```bash
+gpg --verify SHA256SUMS.gpg SHA256SUMS
+```
+
+### If you don't have the keys...
+
+If there is no public key for Ubuntu already present, you will get an error message similar to the following:
 
 ```bash
-gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys "8439 38DF 228D 22F7 B374 2BC0 D94A A3F0 EFE2 1092" "C598 6B4F 1257 FFA8 6632 CBA7 4618 1433 FBB7 5451"
```
Comment (degville) [inline tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md]: *no-highlight*?

File: tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md [hunk 4]
```diff
@@ -2,91 +2,172 @@
 id: tutorial-how-to-verify-ubuntu
 summary: Verifying your ISO helps insure the data integrity and authenticity of your download.
 categories: desktop
-tags: tutorial,installation,verify,ubuntu
+tags: tutorial,installation,verify,ubuntu,18.04
```
Comment (Chrissy4605) [inline tutorials/desktop/how-to-verify-ubuntu/how-to-verify-ubuntu.md]: I could really use a tutorial for18.04

**Pull request merged**